daemon/CMakeLists.txt

```diff
@@ -54,8 +54,12 @@ foreach(P daemon)
     endif()
 
     if (WITH_SYSTEMD)
+        pkg_get_variable(SYSTEMD_SYSTEM_UNIT_DIR
+            systemd systemdsystemunitdir
+            DEFINE_VARIABLES prefix=)
+        string(SUBSTRING ${SYSTEMD_SYSTEM_UNIT_DIR} 1 -1 SYSTEMD_SYSTEM_UNIT_DIR)
```

Comment:
What's that first char that you're removing (I don't have systemd to check)? Is it always going to be there?